js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetCheckDetailModal.tsx

```diff
@@ -281,15 +289,15 @@ export function MigrationRequired() {
               A data migration is required to use asset checks. Run{' '}
               <Mono>dagster instance migrate</Mono>.
             </Body2>
-            <Box
```

Comment:
this should probably be shared

Reply:
yeah I'll do that in a follow up when we add it back